pyfar/classes/audio.py

```diff
@@ -1391,7 +1392,8 @@ def _assert_match_for_arithmetic(data: tuple, domain: str, division: bool,
                     if n_samples != d.n_samples:
                         raise ValueError(
                             "The number of samples does not match.")
-                    fft_norm = _match_fft_norm(fft_norm, d.fft_norm, division)
+                    fft_norm = _match_fft_norm(
+                        fft_norm, d.fft_norm, division, n_audio_objects)
```

Comment:
```suggestion
                    fft_norm = d.fft_norm if n_audio_objects < 2 else \
                        _match_fft_norm(fft_norm, d.fft_norm, division)
```
I would not change the behaviour of _match_fft_norm, instead i would just call it, if we have more then 1 audio object.